app/views/TrusteesNameView.scala.html

```diff
@@ -22,7 +22,7 @@
     formHelper: FormWithCSRF
 )
 
-@(form: Form[_], mode: Mode, index : Int)(implicit request: Request[_], messages: Messages)
+@(form: Form[_], mode: Mode, index : Int, heading: String)(implicit request: Request[_], messages: Messages)
 
 @main_template(
     title = s"${errorPrefix(form)} ${messages("trusteesName.title")}"
```

Comment:
This will need to use `heading` as well to ensure the correct page h1 is shown in the browser tab title.